chia/types/blockchain_format/program.py

```diff
@@ -74,6 +74,25 @@ def __bytes__(self) -> bytes:
     def __str__(self) -> str:
         return bytes(self).hex()
 
+    def at(self, position: str) -> "Program":
+        """
+        Take a string of only `f` and `r` characters and follow the corresponding path.
+
+        Example:
+
+        `assert Program.to(17) == Program.to([10, 20, 30, [15, 17], 40, 50]).at("rrrfrf")`
+
+        """
+        v = self
+        for c in position.lower():
+            if c not in "fr":
+                raise ValueError(f"`at` got illegal character `{c}`. Only `f` & `r` allowed")
+            if c == "f":
+                v = v.first()
+            else:
+                v = v.rest()
```

Comment:
Not a big deal, but the duplicate definition of the set of valid commands can be avoided.

```suggestion
            if c == "f":
                v = v.first()
            elif c == "r":
                v = v.rest()
            else:
                raise ValueError(f"`at` got illegal character `{c}`. Only `f` & `r` allowed")
```

Reply:
Accepted & rebased (ie. I'm stealing your git blame)